Volume_Coupling/Generic_volVectorField.H

```diff
@@ -0,0 +1,40 @@
+#ifndef GENERIC_VOLVECTORFIELD_H
+#define GENERIC_VOLVECTORFIELD_H
+
+#include "CouplingDataUser.H"
+
```

Comment:
@Alphaoo1 I guess you need to add the appropriate include here `fixedValuePointPatchFields.H` or similar

Reply:
Seems to be a step in the right direction, but requires more changes. With this, I get:

> Volume_Coupling/Generic_volVectorField.C:77:40: error: invalid initialization of reference of type ‘Foam::fixedValuePointPatchVectorField&’ {aka ‘Foam::fixedValuePointPatchField<Foam::Vector<double> >&’} from expression of type ‘Foam::fvPatchField<Foam::Vector<double> >’

Reply:
@Alphaoo1 Were you able to solve this?

Reply:
@uekerman  Sorry for the late reply.
No I was not able to solve this issue.
I try to compile the adapter locally but get compilation errors.